docs/Administrator-Guide/Managing-Snapshots.md

```diff
@@ -55,7 +46,7 @@ fails. This to ensure that the snapshot is in a consistent state.
 ### Snapshot creation
 
 ```console
-gluster snapshot create <snapname> <volname> [no-timestamp] [description <description>] [force]
+gluster snapshot create <snapname> <volname> [no-timestamp] [description <description>]
```

Comment:
Force option isn't removed from snapshot this change is invalid https://github.com/gluster/glusterfs/blob/devel/cli/src/cli-cmd-snapshot.c#L73

Reply:
Actually the force option is redundant and the PR to remove the force option is under review. https://github.com/gluster/glusterfs/pull/2110

Reply:
Ack

Reply:
The force option is now removed and the PR is merged. PR: https://github.com/gluster/glusterfs/pull/2110